python/desc/sims/GCRCatSimInterface/InstanceCatalogWriter.py

```diff
@@ -192,6 +276,15 @@ def write_catalog(self, obsHistID, out_dir='.', fov=2):
 
             gal_cat.write_catalog(os.path.join(out_dir, gal_name), chunk_size=100000,
                                   write_header=False)
+        
+        # SN instance catalogs
+        for i, snpop in enumerate(snpopcsvs):
+            phosimcatalog = snphosimcat(snpop, tableName=names[i],
+                                        obs_metadata=obs_md, objectIDtype=i+42)
+
+            snOutFile = name +'_cat_{}.txt'.format(obsHistID)  
+            phosimcatalog.write_catalog(os.path.join(out_dir, snOutFile),
+                                        chunk_size=10000, write_header=False)
 
         if self.imsim_catalog:
 
```

Comment:
line 296 below should be
```
         for orig_name in object_names:
```
so that the SN files get gzipped.

Reply:
I don't see the `object_names` variable somewhere.

Reply:
Sorry, should be the `object_catalogs` list made above.

Reply:
Done